installer/updater/Makefile

```diff
@@ -46,9 +39,11 @@ updater: updater.o $(OBJS)
 	@echo "[ LINK ] $@"
 	$(CXX) $(CPPFLAGS) -fPIC -o 'updater' $^ \
     $(FRAMEBUFFER_LIBS) \
-    $(CURL_LIBS) \
-    $(BORINGSSL_LIBS) \
     -L/system/vendor/lib64 \
+    -L/usr/lib \
+    -lcurl \
```

Comment:
why didn't the ones in phonelibs work? now it's linking against different libs, which are dynamic